packages/dawai-microservice/src/decorators/query.decorator.ts

```
@@ -0,0 +1,21 @@
+import { metadataStorage } from './metadata.storage';
+import { ParameterType } from './parameter.options';
+
+export function Query(key?: string): ParameterDecorator {
+  return (target: Object, propertyKey: string | symbol | undefined, parameterIndex: number) => {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)